{'name': 'test', 'msg': '디버그 메세지 입니다', 'args': (), 'levelname': 'DEBUG', 'levelno': 10, 'pathname': 'C:\\dev\\workspace_python\\week2\\ex_db\\mylogger.py', 'filename': 'mylogger.py', 'module': 'mylogger', 'exc_info': None, 'exc_text': None, 'stack_info': None, 'lineno': 27, 'funcName': '<module>', 'created': 1741765677.9725556, 'msecs': 972.5556373596191, 'relativeCreated': 0.9970664978027344, 'thread': 10356, 'threadName': 'MainThread', 'processName': 'MainProcess', 'process': 7780, 'message': '디버그 메세지 입니다'}(asctime)s - test -DEBUG - 디버그 메세지 입니다
{'name': 'test', 'msg': '정보 메세지입니다', 'args': (), 'levelname': 'INFO', 'levelno': 20, 'pathname': 'C:\\dev\\workspace_python\\week2\\ex_db\\mylogger.py', 'filename': 'mylogger.py', 'module': 'mylogger', 'exc_info': None, 'exc_text': None, 'stack_info': None, 'lineno': 28, 'funcName': '<module>', 'created': 1741765677.9725556, 'msecs': 972.5556373596191, 'relativeCreated': 0.9970664978027344, 'thread': 10356, 'threadName': 'MainThread', 'processName': 'MainProcess', 'process': 7780, 'message': '정보 메세지입니다'}(asctime)s - test -INFO - 정보 메세지입니다
{'name': 'test', 'msg': '경고메세지', 'args': (), 'levelname': 'WARNING', 'levelno': 30, 'pathname': 'C:\\dev\\workspace_python\\week2\\ex_db\\mylogger.py', 'filename': 'mylogger.py', 'module': 'mylogger', 'exc_info': None, 'exc_text': None, 'stack_info': None, 'lineno': 29, 'funcName': '<module>', 'created': 1741765677.9725556, 'msecs': 972.5556373596191, 'relativeCreated': 0.9970664978027344, 'thread': 10356, 'threadName': 'MainThread', 'processName': 'MainProcess', 'process': 7780, 'message': '경고메세지'}(asctime)s - test -WARNING - 경고메세지
{'name': 'test', 'msg': '심각한 오류', 'args': (), 'levelname': 'ERROR', 'levelno': 40, 'pathname': 'C:\\dev\\workspace_python\\week2\\ex_db\\mylogger.py', 'filename': 'mylogger.py', 'module': 'mylogger', 'exc_info': None, 'exc_text': None, 'stack_info': None, 'lineno': 30, 'funcName': '<module>', 'created': 1741765677.9725556, 'msecs': 972.5556373596191, 'relativeCreated': 0.9970664978027344, 'thread': 10356, 'threadName': 'MainThread', 'processName': 'MainProcess', 'process': 7780, 'message': '심각한 오류'}(asctime)s - test -ERROR - 심각한 오류
{'name': 'test', 'msg': '디버그 메세지 입니다', 'args': (), 'levelname': 'DEBUG', 'levelno': 10, 'pathname': 'C:\\dev\\workspace_python\\week2\\ex_db\\mylogger.py', 'filename': 'mylogger.py', 'module': 'mylogger', 'exc_info': None, 'exc_text': None, 'stack_info': None, 'lineno': 27, 'funcName': '<module>', 'created': 1741765850.361773, 'msecs': 361.7730140686035, 'relativeCreated': 1.0445117950439453, 'thread': 8460, 'threadName': 'MainThread', 'processName': 'MainProcess', 'process': 1880, 'message': '디버그 메세지 입니다'}(asctime)s - test -DEBUG - 디버그 메세지 입니다
{'name': 'test', 'msg': '정보 메세지입니다', 'args': (), 'levelname': 'INFO', 'levelno': 20, 'pathname': 'C:\\dev\\workspace_python\\week2\\ex_db\\mylogger.py', 'filename': 'mylogger.py', 'module': 'mylogger', 'exc_info': None, 'exc_text': None, 'stack_info': None, 'lineno': 28, 'funcName': '<module>', 'created': 1741765850.361773, 'msecs': 361.7730140686035, 'relativeCreated': 1.0445117950439453, 'thread': 8460, 'threadName': 'MainThread', 'processName': 'MainProcess', 'process': 1880, 'message': '정보 메세지입니다'}(asctime)s - test -INFO - 정보 메세지입니다
{'name': 'test', 'msg': '경고메세지', 'args': (), 'levelname': 'WARNING', 'levelno': 30, 'pathname': 'C:\\dev\\workspace_python\\week2\\ex_db\\mylogger.py', 'filename': 'mylogger.py', 'module': 'mylogger', 'exc_info': None, 'exc_text': None, 'stack_info': None, 'lineno': 29, 'funcName': '<module>', 'created': 1741765850.361773, 'msecs': 361.7730140686035, 'relativeCreated': 1.0445117950439453, 'thread': 8460, 'threadName': 'MainThread', 'processName': 'MainProcess', 'process': 1880, 'message': '경고메세지'}(asctime)s - test -WARNING - 경고메세지
{'name': 'test', 'msg': '심각한 오류', 'args': (), 'levelname': 'ERROR', 'levelno': 40, 'pathname': 'C:\\dev\\workspace_python\\week2\\ex_db\\mylogger.py', 'filename': 'mylogger.py', 'module': 'mylogger', 'exc_info': None, 'exc_text': None, 'stack_info': None, 'lineno': 30, 'funcName': '<module>', 'created': 1741765850.361773, 'msecs': 361.7730140686035, 'relativeCreated': 1.0445117950439453, 'thread': 8460, 'threadName': 'MainThread', 'processName': 'MainProcess', 'process': 1880, 'message': '심각한 오류'}(asctime)s - test -ERROR - 심각한 오류
{'name': 'test', 'msg': '디버그 메세지 입니다', 'args': (), 'levelname': 'DEBUG', 'levelno': 10, 'pathname': 'C:\\dev\\workspace_python\\week2\\ex_db\\mylogger.py', 'filename': 'mylogger.py', 'module': 'mylogger', 'exc_info': None, 'exc_text': None, 'stack_info': None, 'lineno': 27, 'funcName': '<module>', 'created': 1741768630.3759358, 'msecs': 375.93579292297363, 'relativeCreated': 1.4004707336425781, 'thread': 3404, 'threadName': 'MainThread', 'processName': 'MainProcess', 'process': 11280, 'message': '디버그 메세지 입니다'}(asctime)s - test -DEBUG - 디버그 메세지 입니다
{'name': 'test', 'msg': '정보 메세지입니다', 'args': (), 'levelname': 'INFO', 'levelno': 20, 'pathname': 'C:\\dev\\workspace_python\\week2\\ex_db\\mylogger.py', 'filename': 'mylogger.py', 'module': 'mylogger', 'exc_info': None, 'exc_text': None, 'stack_info': None, 'lineno': 28, 'funcName': '<module>', 'created': 1741768630.3759358, 'msecs': 375.93579292297363, 'relativeCreated': 1.4004707336425781, 'thread': 3404, 'threadName': 'MainThread', 'processName': 'MainProcess', 'process': 11280, 'message': '정보 메세지입니다'}(asctime)s - test -INFO - 정보 메세지입니다
{'name': 'test', 'msg': '경고메세지', 'args': (), 'levelname': 'WARNING', 'levelno': 30, 'pathname': 'C:\\dev\\workspace_python\\week2\\ex_db\\mylogger.py', 'filename': 'mylogger.py', 'module': 'mylogger', 'exc_info': None, 'exc_text': None, 'stack_info': None, 'lineno': 29, 'funcName': '<module>', 'created': 1741768630.3759358, 'msecs': 375.93579292297363, 'relativeCreated': 1.4004707336425781, 'thread': 3404, 'threadName': 'MainThread', 'processName': 'MainProcess', 'process': 11280, 'message': '경고메세지'}(asctime)s - test -WARNING - 경고메세지
{'name': 'test', 'msg': '심각한 오류', 'args': (), 'levelname': 'ERROR', 'levelno': 40, 'pathname': 'C:\\dev\\workspace_python\\week2\\ex_db\\mylogger.py', 'filename': 'mylogger.py', 'module': 'mylogger', 'exc_info': None, 'exc_text': None, 'stack_info': None, 'lineno': 30, 'funcName': '<module>', 'created': 1741768630.3759358, 'msecs': 375.93579292297363, 'relativeCreated': 1.4004707336425781, 'thread': 3404, 'threadName': 'MainThread', 'processName': 'MainProcess', 'process': 11280, 'message': '심각한 오류'}(asctime)s - test -ERROR - 심각한 오류
2025-03-12 17:37:16,885 - test - DEBUG - 디버그 메시지입니다
2025-03-12 17:37:16,885 - test - INFO - 정보 메시지입니다
2025-03-12 17:37:16,885 - test - WARNING - 경고 메시지
2025-03-12 17:37:16,885 - test - ERROR - 심각한 오류
